examples/next-js/src/Chakra.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ChakraProvider,
+  cookieStorageManager,
+  localStorageManager,
+} from "@chakra-ui/core"
+
+export const Chakra = ({ children, cookies }) => {
+  return (
+    <ChakraProvider
+      colorModeManager={
+        typeof cookies === "string"
+          ? cookieStorageManager(cookies)
+          : localStorageManager
```

Comment:
ensures that given cookies, will assume you're rendering server side. could also do `... ? cookieStorageManager(cookies) : undefined` and use the implicit fallback to `localStorageManager` but figured this is more explicit